internal/compiler/widgets/native/std-widgets.slint

```diff
@@ -257,3 +257,88 @@ export GridBox := GridLayout {
     spacing: NativeStyleMetrics.layout-spacing;
     padding: NativeStyleMetrics.layout-spacing;
 }
+
+export StandardTableView := Rectangle {
+    callback sort-ascending(int);
+    callback sort-descending(int);
+    
+    out property <int> current-sort-column: -1;
+    in-out property <[TableColumn]> columns;
```

Comment:
```suggestion
    in property <[TableColumn]> columns;
```

I think the property doesn't need to be modified, does it?
The content of the model will be changed, but not the model itself.

Reply:
Yes you're right.

Reply:
I need it for the sorting. Without it I cannot set e.g. `columns[current-sort-column].sort-by-ascending = false;`